### PR TITLE
Homeworks

### PR DESCRIPTION
**Pull request by maria8ch**
HW 1 pull, with all questions completed (to the best of my ability).

### CODE REVIEW

**Review by bsipocz — CHANGES_REQUESTED** [review]
Two minor things:

* Please do remove the notebook from the submission.

`git rm HR_draft.ipynb` should do the trick.

* Please move it under `assignments/homework/`, after you created a directory for you in there.  Do the move with `git mv `.

Then commit the changes, push to github.

**Comment by maria8ch** [comment]
@bsipocz I have made the edits you have requested, hopefully correctly :)

File: assignments/homework/chernyavskaya/497hw1.py [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+
+
+# Challenge: Plotting the solar system potential
+# 1. The gravitational potential around a point source is V=-GM/r where M is the mass of the point and $r$ is the radius from it. 
+#Use what we've seen above to make a plot of the gravitational potential (in MJ/kg) in the solar system between 1 and 50 AU.
+r = np.linspace(1,50, 1000) * u.AU;
+V = ((-const.G * const.M_sun)/r).to(u.MJ / u.kg);
+plt.plot(r, V, color = 'green', linewidth = 3, label='Gravitational potential');
+plt.xlabel('AU', fontsize=12);
+plt.ylabel('MJ/ kg', fontsize=12);
```
Comment (bsipocz) [inline assignments/homework/chernyavskaya/497hw1.py:56]: You don't need these two lines, the point of using Quantities and enabling `quantity_support()` above is that matplotlib is able to handle quantities, including adding the correct labels to the axes.

https://docs.astropy.org/en/stable/visualization/matplotlib_integration.html#plotting-quantities

File: assignments/homework/chernyavskaya/497hw1.py [hunk 2]
```diff
@@ -0,0 +1,144 @@
+import numpy as np
+import astropy.units as u
+from astropy.units import imperial as imp
+import astropy.constants as const
+from astropy.time import Time
+from astropy.coordinates import SkyCoord, FK5, EarthLocation, AltAz, get_sun
+from astroplan.plots import plot_sky, plot_altitude
+import astroplan as ap
```
Comment (bsipocz) [inline assignments/homework/chernyavskaya/497hw1.py:8]: no big deal, but it's a nice practice to only import things you use. 
If you work in an editor or an IDE (as opposed to the browser with notebooks), then there are nice plugins for checking codestyle and will notify you about these.

Here are the python style recommendations (often just called PEP8): https://www.python.org/dev/peps/pep-0008/
(but not everything in pep8 make sense!)

And tools such as `flake8`, `pycodestyle`, etc would do the codecheck, and tools like `autopep8` would apply some automated fixes.

File: assignments/homework/chernyavskaya/497hw1.py [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+plt.show();
+
+
+
+# Challenge
+# 1. Construct a time object for the current time (note that there is a shortcut for this)
+now = Time.now();
+
+# 2. Find a way to get an ISO 8601 string for the current time, optionally with 6 decimal places
+now_string = Time(now, format='iso', precision=6);
```
Comment (bsipocz) [inline assignments/homework/chernyavskaya/497hw1.py:68]: you don't necessarily need to create a  new `Time` here, you can change the precision of the previous one as:

`now.precision = 6`

File: assignments/homework/chernyavskaya/497hw1.py [hunk 4]
```diff
@@ -0,0 +1,144 @@
...
+    b = np.random.uniform(-1, 1, 30) * u.deg
+    random = SkyCoord(l, b, frame = 'galactic')
+    random = random.transform_to(FK5(equinox='J2020'))
+    print('3. Plot')
+    plt.scatter(random.ra, random.dec, marker='*', color='blue', edgecolors='black', s=700);
+    plt.title("Random positions in the Galactic plane", fontsize=20)
+    plt.xlabel("l, deg", fontsize=14);
+    plt.ylabel("b, deg", fontsize=14);
+    plt.show()
+    return
```
Comment (bsipocz) [inline assignments/homework/chernyavskaya/497hw1.py:99]: No need to return `None`, it is the default behaviour of Python

**Review by bsipocz — APPROVED** [review]
I've left a few minor tips and advices, otherwise, the solutions are correct and the code is nice and clean.

Thank you!

**Pull request merged**